app/models/sipity/workflow.rb

```diff
@@ -10,10 +10,16 @@ class Workflow < ActiveRecord::Base
     has_many :workflow_states, dependent: :destroy, class_name: 'Sipity::WorkflowState'
     has_many :workflow_actions, dependent: :destroy, class_name: 'Sipity::WorkflowAction'
     has_many :workflow_roles, dependent: :destroy, class_name: 'Sipity::WorkflowRole'
+    has_many :permission_templates, dependent: :restrict_with_exception, class_name: 'Hyrax::PermissionTemplate'
 
     DEFAULT_INITIAL_WORKFLOW_STATE = 'new'.freeze
     def initial_workflow_state
       workflow_states.find_or_create_by!(name: DEFAULT_INITIAL_WORKFLOW_STATE)
     end
+
+    def self.default_workflow
+      return if Sipity::Workflow.none?
```

Comment:
I'm not excited about returning `nil` if there aren't any workflows. It is likely to create upstream errors.

@jcoyne Is it a reasonable assumption that there exists a default workflow within the application? If so, I'd like this to remove this line and allow the exception from `first!` to bubble up.